OrbitCaptureClient/CaptureListener.h

```diff
@@ -2,8 +2,8 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-#ifndef ORBIT_GL_CAPTURE_LISTENER_H_
-#define ORBIT_GL_CAPTURE_LISTENER_H_
```

Comment:
For new components please follow the structure as it is in OrbitBase - please include files under indlude/<component_name>/File.h

Reply:
The idea is to use inculde/ directory for public includes and source directory for private ones. This way the file which is not supposed to be used from the outside will not be on the include path.

Reply:
I have fixed this, thanks for letting me know and the explanation